ui/nuxeo-data-table/data-table-checkbox.js

```diff
@@ -17,6 +17,7 @@ import '../widgets/nuxeo-checkmark.js';
             display: flex;
             align-items: center;
             justify-content: center;
+            @apply --nuxeo-data-table-checkbox;
```

Comment:
Same here, it should be something like:

```
@apply --iron-data-table-checkbox;
```
or
```
@apply --data-table-checkbox;
```

Reply:
I believe `--iron-data-table-checkbox` is the best option here too.